addons/binding/org.openhab.binding.neeo/src/main/java/org/openhab/binding/neeo/internal/models/NeeoRoom.java

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.neeo.internal.models;
+
+/**
+ * The model representing an Neeo Room (serialize/deserialize json use only).
+ *
+ * @author Tim Roberts - Initial contribution
+ */
+public class NeeoRoom {
+
+    /** The name. */
+    private final String name;
+
+    /** The key. */
+    private final String key;
+
+    /** The scenarios. */
+    // private final NeeoDevice[] devices;
+    private final NeeoScenarios scenarios;
+
+    /** The recipes. */
+    private final NeeoRecipes recipes;
+
+    /**
+     * Instantiates a new neeo room.
+     *
+     * @param name the name
+     * @param hasController the has controller
+     * @param key the key
+     * @param scenarios the scenarios
+     * @param recipes the recipes
+     */
+    public NeeoRoom(String name, boolean hasController, String key, NeeoScenarios scenarios, NeeoRecipes recipes) {
```

Comment:
This constructor (and probably other constructors in all classes that are built from JSON) seems unused.